### ISSUE TITLE
This is more like a question..

### ISSUE THREAD

**Issue by justinjohn0306** [fei-hdu/ca-gan#6]
can we do the process inverse?
Like can it convert a sketch into an image(human)?

**Comment by fei-aiart** [comment]
yes. the corresponding results have been reported in the corresponding paper.
